nawab.py

```diff
@@ -43,25 +44,38 @@ def wrapper(func, args, res):
 
 
 def main():
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("-r", "--retweet", help="Retweet all tweets automatically, doesn't spawn a telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-t", "--tg_bot", help="Doesn't retweet all automatically, can manually retweet through telegram bot",
+                        action='store_true', required=False)
+    parser.add_argument("-b", "--blacklist",type=str, required=False, help="Blacklist the given username")
+    parser.add_argument("-p", "--path", type=str, required=False,
+                        help="Path where the log files be stored. Note to create directory in that path beforehand.")
+    parser.add_argument('-l', '--level', action='count', default=1)
+    parser.add_argument('-h', '--help', action='help', default=argparse.SUPPRESS,
+                        help='Show this help message and exit.')
+    parser.add_argument('-v', '--version', action='version',
+                        version='%(prog)s 1.0', help="Show program's version number and exit.")
+
+    args = vars(parser.parse_args())
+    ## set manual loglevels
+    level = 40 - (10*args['level']) if args['level'] > 0 else 0
+    
     res = []
     data = pd.read_csv('data.csv')
     default_dir = '/var/log/nawab/'
-
-    u_id = pwd.getpwuid(os.getuid()).pw_name
-
-    if not os.path.exists(default_dir):
-        os.system(("sudo mkdir %s" % (default_dir)))
-
-    ownership_command = "sudo chown %s: %s" % (u_id, default_dir)
-    os.system(ownership_command)
```

Comment:
This should be there. Once the user creates a folder in /var/log, it will still be only accessible by root. We need to change ownership anyhow.